### DISCUSSION TITLE
Welcome to micronucleus Discussions!

### DISCUSSION THREAD

**Discussion by cpldcpu** [micronucleus/micronucleus#200] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by pilotto8** [comment]
Hy. I'm makeing a project in witch I have an ESP8266 that have to transmit a string to an attiny85, witch will then type it on the computer emulating a keyboard. And I'm trying to figure out wath's the best solution to transmit and then write those strings. Any ideas?

**Comment by bhaskar175** [comment]
https://github.com/micronucleus/micronucleus/discussions
